src/extensions/renderer/canvas/drawing-nodes.js

```diff
@@ -114,7 +122,7 @@ CRp.drawNode = function( context, node, shiftToOriginWithBb, drawLabel = true, s
     }
   }
 
-  let drawShape = () => {
+  let drawPath = () => {
```

Comment:
Not a huge deal but this function should probably be named differently, especially since the core render methods are prefixed with the `draw` prefix. Maybe something like `ensurePath` would be more accurate semantically?

Reply:
Sounds good